storey/table.py

```diff
@@ -167,9 +170,13 @@ async def _add_aggregation_by_key(self, key, base_timestamp, initial_data):
         if self._aggregations_read_only and initial_data is None:
             self._set_aggregations_attrs(key, None)
         else:
+            persist_func = None
+            if self._flush_interval_secs is not None and self._flush_interval_secs > 0:
```

Comment:
If `self._flush_interval_secs is None` we don't need this early persistence? How come?